java/src/jmri/jmrix/loconet/LnMultiMeter.java

```diff
@@ -0,0 +1,74 @@
+package jmri.jmrix.loconet;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Provide access to current meter from the Digitrax Evolution Base Station
+ *
+ */
+public class LnMultiMeter extends jmri.implementation.AbstractMultiMeter implements LocoNetListener {
+
+    private SlotManager sm = null;
+    private LnTrafficController tc = null;
+
+    /**
+     * Create a ClockControl object for a Loconet clock
+     *
+     * @param scm - connection memo
+     */
+    public LnMultiMeter(LocoNetSystemConnectionMemo scm) {
+        super(LnConstants.METER_INTERVAL_MS);
+        this.sm = scm.getSlotManager();
+        this.tc = scm.getLnTrafficController();
+        tc.addLocoNetListener(~0, this);
+
+        initTimer();
+    }
+
+    @Override
+    public void message(LocoNetMessage msg) {
+        if (msg.getOpCode() != LnConstants.OPC_EXP_RD_SL_DATA || msg.getElement(1) != 21 || msg.getElement(2) == 249) {
+            return;
+        }
+        log.debug("Found slot 249");
+        // CS Types supported
+        switch (msg.getElement(16)) {
+            case LnConstants.RE_IPL_DIGITRAX_HOST_DCS240:
+            case LnConstants.RE_IPL_DIGITRAX_HOST_DCS210:
+                log.debug("Found Evolution CS Amps[{}] Max[{}]",msg.getElement(6) / 10.0f, (msg.getElement(7) / 10.0f));
+                setCurrent((msg.getElement(6) / 10.0f) / (msg.getElement(7) / 10.0f));
+                break;
+            default:
+                // do nothing
+        }
+    }
+
```

Comment:
This code needs some work, because of how the "Query Mode" feature seems to work when there are more than one Query-mode capable device on LocoNet.  So far as I can tell, there is _no guarantee_ that Query-mode requests _will_ be responded-to in a given order.  If a LocoNet has more than one DCS210/DCS240 (or any combination of same), then you need to do something additional to know exactly which Slot 249 report is from the active command station, and which Slot 248 report(s) is(are) from devices other than the active command station.  I believe that you cannot rely on sequence to determine which is a command station.  I believe that there is at least one bit in Query mode slot 248 report which may indicate whether the device is acting as a command station.  But that does not help much since you are querying slot 249.

Best way I can think of to identify which slot report is "the one" is via serial number information, which appears to be consistently in a couple of bytes of Query Mode slot reports.  Your code would need to know the correct serial number, though!